pkg/guestagent/guestagent_linux.go

```diff
@@ -186,6 +188,24 @@ func (a *agent) collectEvent(ctx context.Context, st eventState) (*api.Event, ev
 	}
 	ev.LocalPortsAdded, ev.LocalPortsRemoved = comparePorts(st.ports, newSt.ports)
 	ev.Time = timestamppb.Now()
+
+	// Get network interfaces info
+	cmd := exec.Command("ip", "-j", "a")
```

Comment:
Does this really work with busybox?

Reply:
It works with the Alpine cloud image from `template://alpine`. It does not work with alpine-lima from `template://alpine-iso`, but can easily be made to work by adding the `iproute2` package to the image.

Reply:
I just used a shell script for apple container, since it "forgot" to update the /etc/hosts file.

`echo "127.0.0.1 localhost" >/etc/hosts`
`echo "$(/sbin/ip -o -4 addr list eth0 | awk '{print $4}' | cut -d/ -f1) $(hostname)" >>/etc/hosts`

Reply:
I thought that Alpine used `ip` from busybox by default, but maybe that is what you are saying above